packages/volto/news/6747.documentation

```diff
@@ -0,0 +1,2 @@
+Updated environment variable for language configuration in backend Docker setup. Changed `LANGUAGE` to `SITE_LANGUAGE` in both Makefile and documentation to align with plone/plone-backend#170 changes. Fixed documentation examples showing correct variable usage for site language configuration.
```

Comment:
@stevepiercy Is it correct to say `documentation` here, since there are also changes to the Makefile?

Reply:
It should be split into two files. You can have multiple change logs with the same PR number. That way they will appear under the appropriate heading in the [release notes](https://6.docs.plone.org/volto/release-notes/).

- `6747.documentation` should pertain to `.md` changes.
- `6747.bugfix` should pertain to the `Makefile` changes.

Once that's done, I'll do another review on these files.